doc/inheritance/conf.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+sys.path.append(os.path.abspath('..'))
+
+from ..conf import *
```

Comment:
Make an exception. I don't see any other way to import all of the config variables and override only the extensions.